Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM nginx:1.23.2
```

Comment:
you can avoid the hardcoding version in the docker file by using ARG.
read here more:

 - https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
 - https://docs.docker.com/engine/reference/builder/#arg

so having a single docker file you can build a number of images  with various combinations of NJS/NGINX:

Then during the building of an image you would:

`docker build --build-arg NGINX_VERSION=1.24.0 .`


here is an example:

```Dockerfile

ARG NGINX_VERSION=1.24.0
ARG NJS_VERSION=0.7.12

FROM nginx:${NGINX_VERSION}

ARG NGINX_VERSION
ARG NJS_VERSION
<...>

# then use it in the RUN

<...> git clone -b ${NJS_VERSION} https://github.com/nginx/njs.git .;

```